src/main/resources/i18n/messages_de.properties

```
@@ -125,6 +125,18 @@ email.dataExportFailedAdmin.githubLink = Link um ein Issue im Artemis GitHub Pro
 email.successfulDataExportCreationsAdmin.title = Angeforderte Datenexporte wurden f?r deine Instanz erfolgreich erstellt
 email.successfulDataExportCreationsAdmin.text = Datenexporte f?r die folgenden Nutzer wurden erfolgreich erstellt als der Job um die Datenexporte zu erstellen zuletzt ausgef?hrt wurde:
 email.successfulDataExportCreationsAdmin.userLogin = \u2022 {0}
+
+# SSH User Settings
+email.sshKeyAdded.title = Neuer SSH-Schlüssel zu Account hinzugefügt
+email.notification.sshKeyAdded.title = Der folgende SSH-Schlüssel wurde zu deinem Benutzerkonto hinzugefügt:
```

```suggestion
email.notification.sshKeyAdded.title = Ein neuer SSH-Schlüssel wurde zu deinem Benutzerkonto hinzugefügt.
```
Die `:` in here was the inconsistency that was bothering me mostly ;)